android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -1046,4 +1046,14 @@
 		"requiredTech": "Nanotechnology",
 		"uniques": ["Spaceship part", "Triggers a global alert upon completion", "Cannot be purchased"]
 	}
+	
+	// All Era's
+	
+	{
+		"name": "Utopia Project",
+		"cost": 1500,
+		"isNationalWonder": true,
+		"uniques": ["Hidden until [5] social policy branches have been completed", "Triggers a global alert upon build start", 
+			"Triggers a Cultural Victory upon completion", "Hidden when cultural victory is disabled"]
```

Comment:
I think that "Triggers a Cultural Victory upon completion" is enough - when you see that, hide when disabled - and we don't need a separate unique for "hidden when victory disabled".

Reply:
Isn't that actually an assumption that may not always be true in different mods? Maybe you want your mod to have an extensive path of multiple buildings that should be build to win a cultural victory, and disabling the first improves the AI so that it won't go that route? This way it is more modular, although I agree that it is cluttering space a bit more.

Another reason to do this, would be to template it to "Hidden when [] victory is disabled", so that we can hide all spaceship parts & the apollo program for the science victory in the same way.

Reply:
Now THAT is a nice idea! I agree that a templated version could solve other problems